patroni/api.py

```diff
@@ -288,6 +288,10 @@ def do_GET_metrics(self):
         metrics.append("# TYPE patroni_postgres_timeline counter")
         metrics.append("patroni_postgres_timeline{0} {1}".format(scope_label, postgres.get('timeline', 0)))
 
+        metrics.append("# HELP patroni_dcs_last_seen Epoch timestamp when DCS was last contacted successfully by Patroni.")
```

Comment:
Flake8 doesn't like lines longer than 120 symbols: https://github.com/zalando/patroni/pull/2051/checks#step:5:8

Reply:
I've fixed that as well now. Sorry, I should've checked that myself.